staging/vhost-device-gpu/src/main.rs

```diff
@@ -51,10 +47,7 @@ impl TryFrom<GpuArgs> for GpuConfig {
 
 fn start_backend(config: GpuConfig) -> Result<()> {
     info!("Starting backend");
-    let backend = Arc::new(RwLock::new(
-        VhostUserGpuBackend::new(config.clone()).map_err(Error::CouldNotCreateBackend)?,
-    ));
-
+    let backend = VhostUserGpuBackend::new(config.clone()).map_err(Error::CouldNotCreateBackend)?;
```

Comment:
IMHO handling this on a global level like the previous code helps to keep the scope of the lock as minimal as possible. you don't need to add all that locking mechanism.

Reply:
I don't understand what you mean by the "scope" of the lock. I would say this lock on the global level has a bigger "scope" because all methods are under the lock, than that lock being inside only for some things. Anyway I tried to explain the previous comment why I am doing this.